src/MCMCChains.jl

```diff
@@ -12,7 +12,7 @@ using SpecialFunctions
 using Formatting
 import StatsBase: autocov, counts, sem, AbstractWeights,
     autocor, describe, quantile, sample, summarystats, cov
-
+using MLJModelInterface
```

Comment:
Since MLJModelInterface is specified explicitly in almost all (apart from two?) places, it might make sense to only import it:
```suggestion
import MLJModelInterface
```
IMO this also helps to quickly figure out which methods and types are actually from MLJModelInterace.